app/controllers/users.js

```diff
@@ -6,10 +6,10 @@ module.exports.signUp = (req, res, next) => {
   try {
     signUpValidator.validate(req.body);
   } catch (error) {
-    throw error;
+    return next(error);
   }
 
-  usersService
+  return usersService
```

Comment:
This return is not needed btw, since express handles the middlewares flow with next and res.send instead of returns, but it does not break anything either so ¯\\_(ツ)_/¯